nvflare/app_opt/pt/model_persistence_format_manager.py

```diff
@@ -115,20 +116,29 @@ def to_persistence_dict(self) -> dict:
                 persistence_dict[k] = v
         return persistence_dict
 
-    def update(self, ml: ModelLearnable):
+    def update(self, ml: Union[ModelLearnable, FLModel]):
```

Comment:
I don't think this is the right approach. We should keep the current PTModelPersistenceFormatManager the way it is, and then extends it to a subclass to accept FLModel type.

Or we could build a new class to wrap around PTModelPersistenceFormatManager. This new class only accepts FLModel, but turn everything to ModelLearnable behind the scene.

Reply:
Restored `PTModelPersistenceFormatManager` and moved the conversion from `FLModel` to `ModelLearnable` to the new `ModelController`.